docs/xaml/hot-reload.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "XAML Hot Reload for .NET MAUI"
+description: "Learn how to reload changes to your .NET MAUI XAML file instantly on your running app, so you don't have to rebuild your .NET MAUI project after every XAML change."
+ms.date: 06/23/2022
+---
+
+# XAML Hot Reload for .NET MAUI
+
+.NET Multi-platform App UI (.NET MAUI) XAML Hot Reload is a Visual Studio feature that enables you to view the result of XAML changes in your running app, without having to rebuild your project. Without XAML Hot Reload, you have to build and deploy your app every time you want to view the result of a XAML change.
```

Comment:
Is it only supported on DEBUG builds?

Reply:
Yes. Moreover, it will only run when the debugger is attached (So, we don't support Ctrl-F5 just yet(